src/notebooks/controllers/kernelSource/kernelSourceCommandHandler.ts

```diff
@@ -32,7 +33,25 @@ export class KernelSourceCommandHandler implements IExtensionSyncActivationServi
             return;
         }
         this.disposables.push(
-            notebooks.registerKernelSourceActionProvider('jupyter-notebook', {
```

Comment:
This was an oversight, forgot to remove the hardcoding I added for the prototype.